lessons/Data-Science/ai-decision-trees/langs/en-us/pages/comparing-trees.adoc

```diff
@@ -0,0 +1,60 @@
+= Comparing Decision Trees
+
+Use the two decision trees (below) to respond to the prompts.
+
+[cols="^.^1,^.^1", stripes="none"]
+|===
+
+|
+
+@image{../images/tree3.png, 300}
+
+|
+
+@image{../images/tree4.png, 290}
+
+|===
+
+@n What do you Notice about the two decision trees above?
+
+@fitb{}{@ifsoln{The trees are different because the "not flat" branch of the tree on the right terminates with "mug or cup".}}
+
+@fitb{}{}
+
+@fitb{}{}
+
+@n What do you Wonder about the two decision trees?
+
+@fitb{}{}
+
+@fitb{}{}
+
+@fitb{}{}
+
+@n The tree on the @fitbruby{10em}{@ifsoln{left}}{[left / right]} is more accurate, while the tree on the @fitbruby{10em}{@ifsoln{right}}{[left / right]} is more efficient.
```

Comment:
I don't see "accuracy" or "efficiency" defined before kids are expected to complete this worksheet. How will kids know what they mean? ("accuracy" isn't even _used_ in the lesson before this worksheet!)

Reply:
I could use help defining these terms and making sure this conversation makes sense.

I didn't really encounter this discussion elsewhere during my research - it came from my brain - so I don't feel terribly confident in it.